### PR TITLE
Demos update

### PR DESCRIPTION
**Pull request by otraore**
Fixes: #361 and #362

### CODE REVIEW

File: demos/mouse-follow/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Mouse Follow Demo
+
+## What does it do?
+It demonstrates how one implement a sprite that will follow the user's mouse movements.
```
Comment (EtienneBruines) [inline demos/mouse-follow/README.md]: > how one implement**s**

File: demos/mouse-follow/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## What are important aspects of the code?
+These things are key in this demo:
+
+* `FollowSystem`, which implements an update method that set's the space component(sprite)'s position to that of the current mouse position. Like so:
```
Comment (EtienneBruines) [inline demos/mouse-follow/README.md]: > that set**s**

**Comment by otraore** [comment]
Up for review

File: demos/mouse-follow/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Mouse Follow Demo
+
+## What does it do?
+It demonstrates how one implements a sprite that will follow the user's mouse movements.
+
+## What are important aspects of the code?
+These things are key in this demo:
+
+* `FollowSystem`, which implements an update method that sets the space component(sprite)'s position to that of the current mouse position. Like so:
```
Comment (EtienneBruines) [inline demos/mouse-follow/README.md]: *space component* should be written as `SpaceComponent`.

**Comment by EtienneBruines** [comment]
Reviewed. Feel free to merge after changes are made :smile:

**Pull request merged**